src/core/crypto_key_mgmt.c

```diff
@@ -65,22 +69,28 @@ int32_t Crypto_Key_OTAR(void)
             mc_log.blk[log_count].emv[3] = 0x41;
             mc_log.blk[log_count++].em_len = 4;
         }
+#ifdef DEBUG
         printf(KRED "Error: MKID is not valid! \n" RESET);
+#endif
         status = CRYPTO_LIB_ERROR;
         return status;
     }
 
-    for (count = 2; count < (2 + IV_SIZE); count++)
+    for (count = 2; count < (2 + 12); count++)
```

Comment:
Is it safe to assume IV_SIZE is static at 12 now?

Reply:
To be referenced in new issue